public_dropin_notebook_environments/python39_notebook/env_info.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "6386dc1159c606b0d8beddc7",
+  "name": "[DataRobot] Python 3.9 Notebook Drop-In",
```

Comment:
@mx-iao Please review. This will be the information seen in Custom Envs UI. See screenshot.

<img width="1054" alt="image" src="https://user-images.githubusercontent.com/87331889/205173481-f5bf8851-0379-4b15-b9e5-e4c99002f4ab.png">